### PR TITLE
Add centralised logging

### LINKED ISSUE #65: [Feature] Add logging and monitoring to Kubernetes deployment
Add option to include the deployment of monitoring with prometheus and grafana, and logging with kibana to TyphonDL Creation Wizard (see [ote-usecase](https://github.com/typhon-project/use-cases/tree/master/ote-usecase/kubernetes_components))

### PR DESCRIPTION
**Pull request by MarieSaphira**
**USER UPDATE**
Please recreate your DL model if you want to use centralised logging. If you don't want to use logging and don't want to recreate your model, you have to add the following line to your `<DLmodelName>.tdl.properties` file:
```
polystore.logging = false
```
--------------------
Adds centralised logging to the Polystore using Elasticsearch, Kibana and Fluentd (EKF stack).

TODO: update wiki

Closes #65

### CODE REVIEW

**Comment by 5h4m4n** [comment]
I think this looks good.

The only thing we might want to think about or maybe in a future release especially for the kubernetes deployment is where elasticsearch will be running.

In the current setup we are adding elasticsearch within the kubernetes deployment. Elastic is a pretty hungry application with respect to RAM and CPU resources so it might be a good idea to have the opportunity to add elastic in a different nodegroup under the monitoring namespace in kubernetes so that we can isolate its resource allocation.

Also if someone decides to completely separate elasticsearch to sit outside of the kubernetes world should we be able to customize the target host?

File: de.atb.typhondl.acceleo/src/de/atb/typhondl/acceleo/files/generateDockerComposeDeployment.mtl [hunk 1]
```diff
@@ -114,6 +122,47 @@ volumes:
...
+  bind 0.0.0.0
+  port 24224
+</source>
+
+<match *.**>
+  @type copy
+
+  <store>
+    @type elasticsearch
+    host elasticsearch
```
Comment (5h4m4n) [inline de.atb.typhondl.acceleo/src/de/atb/typhondl/acceleo/files/generateDockerComposeDeployment.mtl:149]: Can we customize this field and allow the user to type in a custom host if they are not interested in deploying elasticsearch within the application? If you let me know where I can try to create a pull request on top of this one to add the functionality

**Review by 5h4m4n — APPROVED** [review]
This looks great!

File: de.atb.typhondl.acceleo/src/de/atb/typhondl/acceleo/files/generateDockerComposeDeployment.mtl (hunk 1 above)
Comment (MarieSaphira) [reply de.atb.typhondl.acceleo/src/de/atb/typhondl/acceleo/files/generateDockerComposeDeployment.mtl:149]: Yes we can, good idea :+1: 
I will add an additional field to the wizard

**Pull request merged**